README.md

```diff
@@ -623,10 +624,23 @@ The most interesting metrics are:
 
 * If VictoriaMetrics doesn't work because of certain parts are corrupted due to disk errors,
   then just remove directoreis with broken parts. This will recover VictoriaMetrics at the cost
-  of data loss stored in the broken parts. In the future `vmrecover` tool will be created
+  of data loss stored in the broken parts. In the future, `vmrecover` tool will be created
   for automatic recovering from such errors.
 
+## Roadmap
```

Comment:
Wouldn't it be better if we have roadmap in form of issue so users can subscribe to it?

Reply:
we do